tests/foreman/api/test_contentviewversion.py

```diff
@@ -33,960 +33,1013 @@
-        repo = entities.Repository(gpg_key=gpgkey, product=product, url=FAKE_1_YUM_REPO).create()
-        # sync repository
-        repo.sync()
-        # Create content view
-        content_view = entities.ContentView(organization=org).create()
-        # Associate repository to new content view
-        content_view.repository = [repo]
-        content_view = content_view.update(['repository'])
-        # Publish content view
-        content_view.publish()
-        content_view = content_view.read()
-        # Get published content-view version id
-        self.assertEqual(len(content_view.version), 1)
-        cvv = content_view.version[0].read()
-        self.assertEqual(len(cvv.environment), 1)
-        # Delete the content-view version from selected env
-        content_view.delete_from_environment(cvv.environment[0].id)
-        # Delete the version
-        content_view.version[0].delete()
-        # Make sure that content view version is really removed
-        self.assertEqual(len(content_view.read().version), 0)
-
-    @pytest.mark.upgrade
-    @pytest.mark.tier2
-    def test_positive_delete_non_default(self):
-        """Create content view and publish and promote it to new
-        environment. After that try to disassociate content view from 'Library'
-        and one more non-default environments through 'delete_from_environment'
-        command and delete content view version from that content view.
-
-        :id: 95bb973c-ebec-4a72-a1b6-ad28b66bd11b
-
-        :expectedresults: Content view version deleted successfully
-
-        :CaseLevel: Integration
-
-        :CaseImportance: Critical
-        """
-        org = entities.Organization().create()
-        content_view = entities.ContentView(organization=org).create()
-        # Publish content view
-        content_view.publish()
-        content_view = content_view.read()
-        self.assertEqual(len(content_view.version), 1)
-        self.assertEqual(len(content_view.version[0].read().environment), 1)
-        lce = entities.LifecycleEnvironment(organization=org).create()
-        promote(content_view.version[0], lce.id)
-        cvv = content_view.version[0].read()
-        self.assertEqual(len(cvv.environment), 2)
-        # Delete the content-view version from selected environments
-        for env in reversed(cvv.environment):
-            content_view.delete_from_environment(env.id)
+
+
+@pytest.fixture(scope='module')
+def module_lce_cv(module_org):
+    """Create some entities for all tests."""
+    lce1 = entities.LifecycleEnvironment(organization=module_org).create()
+    lce2 = entities.LifecycleEnvironment(organization=module_org, prior=lce1).create()
+    default_cv = entities.ContentView(organization=module_org, name=DEFAULT_CV).search()
+    # There should be only 1 record returned
+    assert len(default_cv) == 1
+    # There should be only 1 version
+    assert len(default_cv[0].version) == 1
+    default_cv = default_cv[0].version[0]
```

Comment:
Either rename the var to `default_cvv` as you are returning the Content View version.

```suggestion
    default_cvv = default_cv[0].version[0]
```

Else return the CV object and later choose to extract version from it.
```suggestion
    default_cv = default_cv[0]
```